js-apps/meep-frontend/src/js/util/scenario-utils.js

```diff
@@ -417,12 +485,32 @@ export function updateElementInScenario(scenario, element) {
         element,
         FIELD_INT_ZONE_PKT_LOSS
       );
+
+      //if domain name changed, other elements created based on that name must also be updated (default ones)
+      for (var i2 in domain.zones) {
```

Comment:
This seems like an update with potential for making the scenario invalid. Are there many cypress or unit tests to augment the coverage here?

Reply:
cypress cloning test imply clicking in the screen to select an object... was not looked at so no cypress added for that or unit tests